core-client/transports/src/transports/local.rs

```diff
@@ -17,10 +17,11 @@ pub struct LocalRpc<THandler, TMetadata> {
 	receiver: mpsc::UnboundedReceiver<String>,
 }
 
-impl<TMetadata, THandler> LocalRpc<THandler, TMetadata>
+impl<TMetadata, THandler, TMiddleware> LocalRpc<THandler, TMetadata>
 where
 	TMetadata: Metadata,
-	THandler: Deref<Target = MetaIoHandler<TMetadata>>,
+	TMiddleware: Middleware<TMetadata>,
+	THandler: Deref<Target = MetaIoHandler<TMetadata, TMiddleware>> + Unpin,
```

Comment:
Can you explain why `Unpin` is required here? It worked without it before this PR and I don't understand why is it needed now.

Technically, a breaking change too but I guess it's used mostly by `IoHandler`.